wondrous-app/utils/constants.tsx

```diff
@@ -529,7 +529,6 @@ export const PAGES_WITH_NO_SIDEBAR = [
   '/twitter/callback',
   '/twitter/verify-tweet',
   '/reset-password',
-  '/onboarding-dao',
```

Comment:
Have we checked to see where we direct to this link? I know we use this in CollabInvite too

Reply:
nice catch, I'll fix

Reply:
fixed, basically after the user creates the org -> the collab is created, then at the final step we redirect to collab page